src/proxy-ui-api/build.gradle

```diff
@@ -187,6 +187,20 @@ jar {
     classifier = 'plainDependency'
 }
 
+// extract dependency jars from spring boot jar, to be able to package postgresql.jar
+task unpackLibsFromBootJar(type: Copy) {
+    from(zipTree("${buildDir}/libs/proxy-ui-api-1.0.jar")) {
+        include "BOOT-INF/lib/*"
+        eachFile { fcd ->
+            fcd.relativePath = new RelativePath(true, fcd.relativePath.segments.drop(2))
+        }
+        includeEmptyDirs = false
+    }
+    into "$buildDir/unpacked-libs"
+}
+
+jar.finalizedBy(unpackLibsFromBootJar)
+
```

Comment:
suggestion (a bit simpler and also avoids copying unnecessary libs):
```
task copyDeps(type: Copy) {
    into "$buildDir/unpacked-libs"
    from configurations.compileClasspath.find {it.name.startsWith("postgresql") }
}

build.dependsOn copyDeps
```

Reply:
Good suggestion, fixed, much simplified.

Gradle was a bit picky about the place where `copyDeps` should be put. That might explain why I ended up with the original clumsy implementation. `copyDeps` needed to be after `dependencyManagement`, it seems.